src/main/kotlin/com/avail/interpreter/primitive/fibers/P_UnparkFiber.kt

```diff
@@ -44,15 +44,16 @@ import com.avail.descriptor.types.A_Type
 import com.avail.descriptor.types.FiberTypeDescriptor.mostGeneralFiberType
 import com.avail.descriptor.types.FunctionTypeDescriptor.functionType
 import com.avail.descriptor.types.TypeDescriptor.Types.TOP
+import com.avail.interpreter.Primitive
+import com.avail.interpreter.Primitive.Flag.CanInline
+import com.avail.interpreter.Primitive.Flag.CannotFail
+import com.avail.interpreter.Primitive.Flag.HasSideEffect
 import com.avail.interpreter.execution.Interpreter
 import com.avail.interpreter.execution.Interpreter.Companion.resumeFromSuccessfulPrimitive
-import com.avail.interpreter.Primitive
-import com.avail.interpreter.Primitive.Flag.*
-import com.avail.utility.evaluation.Continuation0
 
 /**
  * **Primitive:** Unpark the specified [fiber][FiberDescriptor]. If the
- * [permit ][SynchronizationFlag.PERMIT_UNAVAILABLE] associated with the fiber
+ * [permit&#32;][SynchronizationFlag.PERMIT_UNAVAILABLE] associated with the fiber
```

Comment:
Remove spurious `&#32;`.

Reply:
I cleaned and reflowed all occurrences of this problem. There were three.